jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/GenericErrorSummaryCountCell.java

```diff
@@ -65,10 +73,10 @@ public void render(Context context,
             tooltipClasses.remove(ERROR_PRESENT_STYLE);
         }
 
-        if (viewPresenter.getViewErrorsActionCondition().test(value)) {
+        if (viewPresenter.isUserAuthorizedForPerspective(EXECUTION_ERRORS) && value.getErrorCount() != null && value.getErrorCount() > 0) {
```

Comment:
@bsremac The purpose of the second and third condition is showing a tooltip with the message "Click to view {n} error" when the number of errors is more than 0 or it's not null. Otherwise nothing is shown (in case of zero or null).
This line set the value of the errorCount in the cell, in order to set the value to zero when is null or the correct value in any other cases.
https://github.com/mcivantos-tribalyte/jbpm-wb/blob/5f4fc0f2772210b94a0d683571b3a2acf120b8f2/jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/GenericErrorSummaryCountCell.java#L65

Reply:
@mcivantos-tribalyte I see now where ```errorCount``` is used inside _if - else_ clause.
However, most of my previous comment still holds.
The call to ```value.getErrorCount()``` is already processed in line 65 and if you use ```errorCount``` instead of ```value.getErrorCount()``` in your condition, the current condition can be reduced to:
```
if (viewPresenter.isUserAuthorizedForPerspective(EXECUTION_ERRORS) && errorCount > 0)
```
Furthermore, the satisfaction of the condition ```errorCount > 0``` has already been addressed in line 70 when it comes to tooltip visibility and behavior, so I'm not sure why aren't these two conditional branches merged into one.